code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -44,9 +46,9 @@
 	update_icon()
 
 /obj/item/storage/box/fancy/update_icon(var/itemremoved = 0)
+	total_contents = src.contents.len - itemremoved
 	if(opened) //use the open icon.
 		if(icon_overlays) //whether it uses the overlays/uses its own version.
-			var/total_contents = src.contents.len - itemremoved
 			src.icon_state = "[src.icon_type][src.storage_type][total_contents]"
```

Comment:
if total_contents isn't going to be used anywhere else, we can just stick the calculation in this line

```suggestion
			src.icon_state = "[src.icon_type][src.storage_type][contents.len - itemremoved]"
```